src/Application/Models/Asset.js

```diff
@@ -1,7 +1,7 @@
 import { assetDefaults } from './../../util/constants';
 
 class Asset {
-  constructor(name, price, multiplier, owned, unlocked, investment, revenue) {
+  constructor(name, price, multiplier = 1, owned = 0, unlocked = false, investment = 0, revenue = 0) {
```

Comment:
yeah this is interesting, because it becomes a question of who owns this default information, who's calling the shots.  If it's the responsibility / domain of `StateUtils` to define defaults, put them there.  If it's inherent to the model, put them here.  It doesn't matter all that much, and helps to reduce verbosity to put it here which is nice, but this is the principle that should be considered when making this decision